.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -373,6 +373,10 @@ stages:
       displayName: Upload linux-x64 monitoring home
       artifact: linux-monitoring-home-$(baseImage)
 
+    - publish: $(System.DefaultWorkingDirectory)/shared/symbols
+      displayName: Upload linux-x64 symbols
+      artifact: linux-symbols-home-$(baseImage)
+
```

Comment:
We can do it in a separate PR if you prefer (I'm happy to do it if you like) but we should add this artifact to the release artifacts.

Also, we should add the upload to the arm64 stage too - we're only stripping out the tracer native dll currently, but we probably don't want to lose that info 🙂

Reply:
Yeah that was my concern, I wanted it in there so we can discuss.

I'm happy to have it in another PR and happier if it's amongst the release artifacts..

If you can do it that would be great thx

Reply:
> If you can do it that would be great thx

Will do, I'll take a look tomorrow 👍